packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.Props.ts

```diff
@@ -447,11 +454,6 @@ export interface IMenuItemStyles extends IButtonStyles {
   * Styles for a divider item of a ConextualMenu.
   */
   divider: IStyle;
-
-  /**
```

Comment:
can you leave this even if you don't use it?

Add @deprecated to the comments.

Otherwise it is a break.

Reply:
Sure will do that :)